tests_e2e/tests/agent_publish/agent_publish.py

```diff
@@ -132,7 +132,7 @@ def _prepare_agent_for_rsm_update(self) -> None:
 
         self._verify_agent_reported_supported_feature_flag()
         arch_type = self._ssh_client.get_architecture()
-        request_rsm_update(self._published_version, self._context.vm, arch_type)
+        request_rsm_update(self._published_version, self._context.vm, arch_type, False)
```

Comment:
nit: Could we use keyword arguments on these calls so it's clear what the bool is for?
```suggestion
        request_rsm_update(self._published_version, self._context.vm, arch_type, is_downgrade=False)
```